README.md

```diff
@@ -34,104 +34,55 @@ Getting Started
 
 Ketrew is very flexible and hence may seem difficult to understand and setup at
 first.
-Let's get a minimal workflow running.
+Let's get a minimal setup ready and a workflow running on it.
```

Comment:
"not-production-ready" well depends one what you produce :)

it'a Ketrew + DB + scheduler running on a single host; it can already be useful (next episode on how I run my backups :) ).